themes/scss/infowindow/cartodb-infowindow-default.scss

```diff
@@ -96,17 +100,38 @@
   bottom: 12px;
 }
 
+.CDB-infowindow-fail {
+  color: rgb(243, 86, 60);
```

Comment:
Properties should be ordered display, transform, color, font-size, text-align, text-transform